README.md

```diff
@@ -37,3 +39,24 @@ Lesson plan:
 
 [11 Mapping Marc to Dublin Core](./docs/11_MARC_to_Dublin_Core.html)
 
+## Testing
+
+### Installation
+If you are on debian derivates go install build-essentials:
+```
+apt install build-essential
+```
+Then you can use the Ruby Dependency Management to build all you need:
+```
+bundle install
+```
+Start jekyll like this:
+```
+bundle exec jekyll serve
+```
+If you experience troubles, make sure to not have jekyll installed via you
+package manager. On debian derivates do:
+```
+apt purge jekyll
```

Comment:
Same here: Does this work for anybody without `sudo`? At least it doesn't for me.